securedrop_client/conversation/transcript/items/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .factory import transcribe  # noqa: F401
```

Comment:
@cfm I don't like this module name much, but couldn't think of anything more expressive. Its blasting radius is small, but if you can think of a better way to expose this function, I'm all ears!